crates/core_arch/src/aarch64/neon/generated.rs

```diff
@@ -297,292 +297,320 @@ mod test {
     use std::mem::transmute;
     use stdarch_test::simd_test;
 
+    macro_rules! verify_fn {
```

Comment:
Additional tests would need to live in a different file `mod.rs` for example? This file, being generated will be completely overwritten on updates.